src/pages/fr/core-concepts/project-structure.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Structure du Projet
+description: Apprends à structurer un projet avec Astro.
+---
+
+Votre tout nouveau projet Astro généré à partir de l'assistant de création `create-astro` a déjà quelques des fichiers et dossiers. Les autres fichiers, vous allez les créer vous-même et ajouter à la structure de fichiers d'Astro.
+
+Voici comment un projet Astro est organisé, et les quelques fichiers que vous trouverez dans votre nouveau projet.
+
+## Répertoires et fichiers
+
+Astro influence une façon d'organiser de dossier pour votre projet. Chaque projet Astro doit inclure les répertoires et fichiers suivants :
+
+- `src/*` - Le code source de votre projet (composants, pages, styles, etc.)
+- `public/*` - Tout ce qu'il n'est pas du code, fichiers non-traités (polices d'écritures, icones, etc.)
+- `package.json` - Le manifeste de votre projet.
+- `astro.config.mjs` - Un fichier de configuration d'Astro (optionel).
+
+### Exemple de structure de projet
+
+Un projet Astro assez commun peut ressembler à ça :
+
+```
+├── src/
+│   ├── components/
+│   │   ├── Header.astro
+│   │   └-─ Button.jsx
+│   ├── layouts/
+│   │   └-─ PostLayout.astro
+│   └── pages/
+│   │   ├── posts/
+│   │   │   ├── post1.md
+│   │   │   ├── post2.md
+│   │   │   └── post3.md
+│   │   └── index.astro
+│   └── styles/
+│       └-─ global.css
+├── public/
+│   ├── robots.txt
+│   ├── favicon.svg
+│   └-─ social-image.png
+├── astro.config.mjs
+└── package.json
+
+```
+
+### `src/`
+
+Le dossier src est où se trouve le code source de votre projet. Il comprend :
+
+- [Des composants Pages](/fr/core-concepts/astro-pages/)
+- [Des composants Layouts](/fr/core-concepts/layouts/)
+- [Des composants Astro](/fr/core-concepts/astro-components/)
+- [Des composants Front-end (React, etc.)](/fr/core-concepts/framework-components/)
```

Comment:
"Front-end" or "Frontend"? I have seen it with hyphen and without. Check for consistency.